addHotelProcess.php

```diff
@@ -0,0 +1,67 @@
+<?php
+require 'connection.php';
+
+$jsonText = $_POST["hotelData"];
```

Comment:
You must validate the value before assigning it to the variable.

Reply:
ok